content/Fundamentals.en.md

```diff
@@ -3,59 +3,58 @@
 ## Links
 
 - Author: https://github.com/tshemsedinov
-- Telegram channel: https://t.me/HowProgrammingWorks
-- Tasks: https://github.com/HowProgrammingWorks/Index/blob/master/Exercises.ru.md
+- Telegram: https://t.me/HowProgrammingWorks
+- Excercises: https://github.com/HowProgrammingWorks/Index/blob/master/Exercises.ru.md
 - Glossary: https://github.com/HowProgrammingWorks/Dictionary/blob/master/Fundamentals.ru.md
 
 ## Module SEF1
 
-36 lectures (4 introductory, 8 on the JS language, 18 on fundamental concepts of
-programming, 6 on development tools and process), 8 seminars, 9 laboratory works,
-47 repositories with code examples.
+36 lectures (4 introductory topics, 8 lectures about JavaScript programming language, 18 lectures about fundamental programming concepts, 
+6 lectures about development tools and processes), 8 seminars, 9 practices and assignments, 47 repositories with code examples.
```

Comment:
```suggestion
36 lectures (4 introductory topics, 8 lectures about JavaScript programming
language, 18 lectures about fundamental programming concepts, 6 lectures about
development tools and processes), 8 seminars, 9 practices and assignments, 47
repositories with code examples.
```

Reply:
Max line length is 80, after https://github.com/HowProgrammingWorks/Book/pull/6 command `npm run fmt` will wrap automatically.